docs/entities/transactions/abstract-transaction.schema.json

```diff
@@ -89,11 +89,29 @@
     "post_condition_mode": {
       "$ref": "../post-conditions/post-condition-mode.schema.json"
     },
-    "events": {
-      "type": "array",
-      "description": "List of transaction events",
-      "items": {
-        "$ref": "../transaction-events/transaction-event.schema.json"
+    "events" : {
```

Comment:
This is a significant breaking change for anyone using this API response. I think a good middle-ground is to include `event_count` as a top-level property, and to leave this property as in (we wouldn't get the returned limit and offset values, but I don't think those are very important).

@kyranjamie @aulneau

Reply:
Yes, this is necessary otherwise all previous wallet builds will break.